packages/server/socketHelpers/ConnectionContext.ts

```diff
@@ -37,6 +42,14 @@ class ConnectionContext<T = WebSocket | HttpResponse> {
     this.readyQueue.forEach((thunk) => thunk())
     this.readyQueue.length = 0
   }
+
+  getMid() {
```

Comment:
if a single connection sends 2B messages, our server will crash before we overflow :sweat_smile: 
this is good though, better safe than sorry. it'd be real hard to debug an overflow like this